src/features/game/types/bumpkinItemBuffs.ts

```diff
@@ -147,4 +147,16 @@ export const BUMPKIN_ITEM_BUFF_LABELS: Partial<Record<BumpkinItem, BuffLabel>> =
       boostTypeIcon: powerup,
       boostedItemIcon: ITEM_DETAILS.Banana.image,
     },
+    "Bee Suit": {
+      shortDescription: "+0.1 Honey",
+      labelType: "success",
+      boostTypeIcon: powerup,
+      boostedItemIcon: ITEM_DETAILS.Honey.image,
+    },
+    "Honeycomb Shield": {
+      shortDescription: "Double Honey",
```

Comment:
For simplicity, I would suggest still using +1 Honey

This will help down the line if we do need to keep it additive 👍